db/migrate/20180204235116_add_mooc_to_code_schools.rb

```diff
@@ -0,0 +1,5 @@
+class AddMoocToCodeSchools < ActiveRecord::Migration[5.0]
+  def change
+    add_column :code_schools, :mooc, :boolean
```

Comment:
Let's set a few things in this migration: 

1. default this column to `false`.  That way, all existing records will be set to false, and the frontend can immediately begin filtering data
2. set `null` to `false`.  This will enforce presence, at the database level.

Ping me if you have any questions on setting these!

Reply:
@hpjaj If I validate the new attribute, won't that break our existing CodeSchool database?

Reply:
@leenyburger - It would if we were not also setting a default of `false`.  But you're good, as long as that default is included, as well.